rbc/irtools.py

```diff
@@ -238,6 +239,8 @@ def compile_to_LLVM(functions_and_signatures, target: TargetInfo, debug=False):
     flags.set('no_compile')
     flags.set('no_cpython_wrapper')
 
+    pipeline_class = OmnisciCompilerPipeline
```

Comment:
We should pass `OmnisciCompilerPipeline` from omniscidb.py via `compile_to_LLVM` its `pipeline_class` argument that needs to be introduced.

Reply:
Remove this line.

Reply:
I forgot to remove it